src/main/java/UseCase/playSongInteractor.java

```diff
@@ -0,0 +1,34 @@
+package UseCase;
+
+import Entities.MusicPlayer;
+import Entities.Song;
+import InputBoundary.playSongInputBoundary;
+import InputData.songInputData;
+import OutputBoundary.songOutputBoundary;
+import OutputData.songOutputData;
+
+
+public class playSongInteractor implements playSongInputBoundary {
+
+    private final MusicPlayer mp = MusicPlayer.getInstance();
+    private final songOutputBoundary songPresenter;
+    private final Song song;
+
+    public playSongInteractor(songInputData s, songOutputBoundary songPresenter){
+        this.song = s.getSong();
+        this.songPresenter = songPresenter;
+    }
+
+    public void playAudio() {
```

Comment:
This is where I'm confused -- there is both the playAudio via the playAudioInteractor, and the playAudio via the playSongInteractor. Which should I be interacting with? I assume playAudioInteractor, in which case this function could be deleted. Please let me know if this is incorrect.